### ISSUE TITLE
Brand guidelines cannot be updated

### ISSUE THREAD

**Issue by minrk** [jupyter/design#69]
The current brand guidelines are a PDF export of presumably a presentation, but we don't seem to have the presentation anywhere in an editable format.

I know we set up a different, non-GitHub workflow for designers to contribute, but I can't find documentation of it, or how folks are supposed to propose updates to the brand guidelines right now (#68).

**Comment by Ruv7** [comment]
I'm not sure if this document exists in a format that is readily editable but I will at mention two folks who might know or have suggestions for how to move forward. @ellisonbg @isabela-pf 

The brand guidelines were a project that Cal Poly interns created and maintained for some years but since that program for students no longer exists we don't have an existing process or resource to fulfill this request.

**Comment by Ruv7** [comment]
Of course, something like this page would be ideal but at this point we don't have the resources for it. https://www.linuxfoundation.org/brand-guidelines

**Comment by minrk** [comment]
What do you think is the best way to address #68 in the meantime?

**Comment by ellisonbg** [comment]
The EC is working on setting up a project wide Figma account that everyone can use. I found the .ai fine for the brand guidelines in the meantime if people want to work with that. But once things are in Figma, I think we should use that as the source of truth.

**Comment by echarles** [comment]
> The EC is working on setting up a project wide Figma account that everyone can use.

That would be wonderful. I link here https://github.com/jupyterlab/team-compass/issues/196 opened by @SylvainCorlay as a good example of logo need/discussion, and would like to share the work we are doing to create visual icons that represent the various technical pieces of the Jupyter ecosystem.

The goal is to have standard ways to represent a Jupyter Server, Kernel, Hub, Gateway... so it can be used in a coherent and standard way to communicate in diagrams, web applications the intent and the mean of what is referred to.

You can see the current WIP on https://icons.datalayer.design/?q=jupyter (sources on https://github.com/datalayer/icons).

I guess JupyterLab ui-components can be a natural recipient of such a work. As I favor React.js components, I have started the work in a separated repository to create such react components (not limited to Jupyter btw).

In all case, I am hoping that a standard SVG collection of artefacts to be produced that cover evolving Jupyter pieces in a meaningful way.

Happy to help with that goal if it makes sense!

cc/ @Carreau @mbektas

**Comment by ellisonbg** [comment]
Quick update: The Jupyter Executive Council is working on a proposed plan for this design repo and Figma. Will report back when we have more information and next steps.

**Comment by imjul1an** [comment]
@ellisonbg, is there any update on that proposal? We might consider contributing to the Figma design system for JL.

**Comment by rgaiacs** [comment]
There is any update on the proposal for this repo? Asking because of conversation on https://github.com/jupyterhub/team-compass/issues/526.

cc @choldgraf
